### PR TITLE
[Docs] Extend thoughts on concepts

### PR DESCRIPTION
**Pull request by tehsphinx**
@iHiD: hope this catches the essential parts. Feel free to adjust!

### CODE REVIEW

**Comment by Calamari** [comment]
I love it. Especially the mentioning of links, where to learn about this in the introduction, is very good for newcomers to a language.

File: docs/maintainers/determining-concepts.md [hunk 1]
```diff
@@ -9,3 +9,11 @@ As many languages use the same concepts (but possibly implemented differently),
 It can also be helpful to group related concepts. For example, the Classes, Polymorphism and Inheritance concepts are all object-oriented concepts, which can then be grouped as "Object-oriented concepts".
 
 Filling out the concepts will be iterative, it is hard to get this right immediately. While creating concepts exercises, you'll probably find missing concepts or perhaps want to split out concepts.
+
+## Concept Exercises
+
+Generally there should be only one new concept introduced per concept exercise. There might be exceptions for _trivial_ concepts (like e.g. `int` in C#) that could be introduced in other concept exercises (e.g. `conditionals`).
+It should then get some attention in the introduction to that exercise with links on the topic -- even if its basically just teaching operators `+.*/`.
```
Comment (kotp) [inline docs/maintainers/determining-concepts.md]:
```suggestion
It should then get some attention in the introduction to that exercise with links on the topic -- even if it is basically just teaching operators `+.*/`.
```

File: docs/maintainers/determining-concepts.md [hunk 2]
```diff
@@ -9,3 +9,11 @@ As many languages use the same concepts (but possibly implemented differently),
...
+The focus in the introduction on _trivial_ concepts should be the same as on other concept introductions: How is it done in this language? If there is nothing special about it, a language-agnostic link is helpful.
```
Comment (ErikSchierboom) [inline docs/maintainers/determining-concepts.md]:
```suggestion
The focus in the introduction on _trivial_ concepts should be the same as on other concept introductions: how is it done in this language? If there is nothing special about it, a language-agnostic link is helpful.
```

File: docs/maintainers/determining-concepts.md [hunk 3]
```diff
@@ -9,3 +9,11 @@ As many languages use the same concepts (but possibly implemented differently),
...
+
+Apart from trivial concepts: If a concept exercise requires another concept to solve it, that should be a prerequisite. For example a concept exercise on floating point numbers that requires a `loop` should have a concept exercise on `loops` as prerequisite.
```
Comment (ErikSchierboom) [inline docs/maintainers/determining-concepts.md]:
```suggestion
Apart from trivial concepts, if a Concept Exercise requires another concept to solve it, that should be a prerequisite. For example, a Concept Exercise on floating point numbers that requires using loops should have the `loops` concept as a prerequisite.
```

**Comment by tehsphinx** [comment]
Changes applied. @kotp , @ErikSchierboom

**Comment by iHiD** [comment]
@tehsphinx I've merged my branch into here. Once this goes green, could someone merge please.

**Comment by tehsphinx** [comment]
No idea how to fix those link checkers. As far as I can see all links in this file are ok.

**Comment by iHiD** [comment]
I'll sort it.

**Comment by tehsphinx** [comment]
Sorted now. I merged this: https://github.com/exercism/v3/pull/132

**Comment by iHiD** [comment]
Thanks. I'm also doing https://github.com/exercism/v3/pull/134 for now. We can have a second periodic check for external links.

**Pull request merged**